doc/datasets/index.rst

```diff
@@ -62,6 +62,7 @@ require to download any file from some external website.
    load_linnerud
    load_wine
    load_breast_cancer
+   fetch_california_housing
```

Comment:
This should not appear under "
scikit-learn comes with a few small standard datasets that do not
require to download any file from some external website.
 "